### ISSUE TITLE
[Feedback] Context menu's 

### ISSUE THREAD

**Issue by UnderKoen** [actualbudget/actual#3706]

**Comment by UnderKoen** [comment]
@jfdoming 
> Interactions still occur when a right click menu opens (in a native menu, all interactions are normally disabled)

In most applications and sites the behavious is to be able to do other interactions. I personally like this more as doesn't need an extra interaction to change what you are doing.

**Comment by Chinoman10** [comment]
> [@jfdoming](https://github.com/jfdoming)
> 
> > Interactions still occur when a right click menu opens (in a native menu, all interactions are normally disabled)
> 
> In most applications and sites the behavious is to be able to do other interactions. I personally like this more as doesn't need an extra interaction to change what you are doing.

I agree with this; if the webapp doesn't have a context menu to use for that 'target', then let the system open the default one instead. Disabling the right-click context menu because you have no use for it for that particular scenario only makes the UX more frustrating (I _**really**_ hate webapps that limit mouse use; I literally go out of my way and lose time just to get it running again (with local JS) out of spite).

**Comment by jfdoming** [comment]
> > [@jfdoming](https://github.com/jfdoming)
> > > Interactions still occur when a right click menu opens (in a native menu, all interactions are normally disabled)
> > 
> > 
> > In most applications and sites the behavious is to be able to do other interactions. I personally like this more as doesn't need an extra interaction to change what you are doing.
> 
> I agree with this; if the webapp doesn't have a context menu to use for that 'target', then let the system open the default one instead. Disabling the right-click context menu because you have no use for it for that particular scenario only makes the UX more frustrating (I _**really**_ hate webapps that limit mouse use; I literally go out of my way and lose time just to get it running again (with local JS) out of spite).

Just to be clear, the idea wasn't to disable the native context menu. It was to disable interactivity (e.g. left clicking on a button) when the custom context menu is open. We would still display native context menus
